website/docs/dev_addon_intro.md

```diff
@@ -0,0 +1,142 @@
+---
+id: dev_addon_intro
+title: Addon Dev Intro
+sidebar_label: Addon Intro
+---
+
+:::info
+This page is still under construction 👷🚧🛠️🔜🏗️.
+
+Feel free to report any missing or wrong info.
+:::
+
+## Preface
+
+AYON is an studio platform for Animation, VFX and games.
```

Comment:
this is just repetition and feels like selling AYON even in the docs. let's be a bit more frugal with these :)